packages/cli/src/services/frontend.service.ts

```diff
@@ -29,6 +206,94 @@ export class FrontendService {
 		this.writeStaticJSON('credentials', credentials);
 	}
 
+	async getSettings(): Promise<IN8nUISettings> {
+		const restEndpoint = config.getEnv('endpoints.rest');
+
+		// Update all urls, in case `WEBHOOK_URL` was updated by `--tunnel`
+		const instanceBaseUrl = getInstanceBaseUrl();
+		this.settings.urlBaseWebhook = WebhookHelpers.getWebhookBaseUrl();
+		this.settings.urlBaseEditor = instanceBaseUrl;
+		this.settings.oauthCallbackUrls = {
+			oauth1: `${instanceBaseUrl}/${restEndpoint}/oauth1-credential/callback`,
+			oauth2: `${instanceBaseUrl}/${restEndpoint}/oauth2-credential/callback`,
+		};
+
+		// refresh user management status
+		Object.assign(this.settings.userManagement, {
+			quota: Container.get(License).getUsersLimit(),
```

Comment:
same here

Reply:
In fact, it turns out many of these helpers `isSourceControlLicensed`, `isExternalSecretsEnabled`, etc. are all calls to the license. I'll simplify.